From f0945562b6c84f0cfeb76422d0d5054e70dee2fb Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Fri, 12 Sep 2025 14:02:21 +0200
Subject: [PATCH] arm64: dts: st: use rtc as clock provider on stm32mp215f-dk

Add the "#clock-cells" property in the rtc node.

Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I7edbdd573b6eee31dac4c0bb9c0cbf2eb4f7146d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/490103
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp215f-dk.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp215f-dk.dts b/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
index 0417a02ff9cc..194de2b62ed3 100644
--- a/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
@@ -483,6 +483,7 @@ mdf_endpoint1: endpoint {
 };
 
 &rtc {
+	#clock-cells = <0>;
 	st,lsco = <RTC_OUT1>;
 	pinctrl-0 = <&rtc_out1_pins_a>;
 	pinctrl-names = "default";
-- 
2.53.0

